### DISCUSSION TITLE
How do i get the border around the file manager like in the gif?

### DISCUSSION THREAD

**Discussion by Un1q32** [mcchrish/nnn.vim#162] [Q&A]
This is what it looks like now and its wierd
![Screenshot (58)](https://user-images.githubusercontent.com/57970114/183128350-ae0bcd2b-329d-4ec1-b379-2b180f91425c.png)

**Comment by mcchrish** [comment]
The border uses the highlight `NnnBorder`. Here's the default:
```vim
if hlexists('FloatBorder')
    highlight default link NnnBorder FloatBorder
else
    highlight default link NnnBorder Comment
endif
```
